package.json

```diff
@@ -391,7 +391,8 @@
           "description": "Specifies Lint tool name.",
           "enum": [
             "golint",
-            "gometalinter"
+            "gometalinter",
+            "megacheck"
```

Comment:
Ah! I was looking for a linting warning that staticcheck, gosimple and unused do not give.

The below works

<img width="1016" alt="screen shot 2017-07-16 at 2 03 02 pm" src="https://user-images.githubusercontent.com/16890566/28251333-cdbf62f6-6a2f-11e7-8fe7-679fac0e6659.png">